### ISSUE TITLE
Add ECC Certificate Support to Azure KeyVault PowerShell

### ISSUE THREAD

**Issue by kashifm-msft** [Azure/azure-powershell#9241]
## Description
Azure CLI has the support for managing ECC Certificates but Azure PowerShell does not. Please add.

**Comment by MiYanni** [comment]
@kashifm-msft What kind of management functions are you looking for involving ECC certs?

**Comment by kashifm-msft** [comment]
Azure KeyVault PowerShell already supports Certificates operation. In order to support ECC Certificate, Azure KeyVault PowerShell needs to:

1)	Support Api Version 7.0
2)	Support Curve field (as document at https://docs.microsoft.com/en-us/rest/api/keyvault/createcertificate/createcertificate#jsonwebkeycurvename)

**Comment by kashifm-msft** [comment]
It appears that version 5.1.2 supports it.

**Comment by kashifm-msft** [comment]
Reopening ...
While the version 5.1.2 supports Api Version 7.0, it does not support the Curve field  (as document at https://docs.microsoft.com/en-us/rest/api/keyvault/createcertificate/createcertificate#jsonwebkeycurvename)

**Comment by MiYanni** [comment]
@kashifm-msft Please work with @cormacpayne as he is generating the upcoming PowerShell cmdlets from swagger.

**Comment by markcowl** [comment]
@RandalliLama Can you take a look at this request to support the necessary parameter for Elliptic Curve Cryptography?

**Comment by johncrim** [comment]
At a minimum, we should be able to store and access externally generated ECC certs in keyvault using powershell commands. Right now this doesn't work.

To clarify, this command fails with EC certs:
```powershell
    Import-AzKeyVaultCertificate `
      -VaultName $keyVaultName `
      -Name $certificateName `
      -FilePath $pfxFile `
      -Password $pfxPassword -Debug
```

**Comment by n0thingness** [comment]
I'm looking for the Azure PowerShell equivalent of the following Azure CLI command:

```
az keyvault key create --name myKeyName --vault-name myVaultName --kty EC --curve P-256K
```

It's rather surprising that both Azure Portal and Azure CLI support this functionality (albeit a [naming discrepancy](https://feedback.azure.com/forums/906355-azure-key-vault/suggestions/38127457-update-key-vault-ec-key-secp256k1-curve-name-to-p)), while Azure PowerShell's Add-AzKeyVaultKey [only supports RSA keys](https://docs.microsoft.com/en-us/powershell/module/az.keyvault/add-azkeyvaultkey?view=azps-2.4.0). I would have assumed that Azure PowerShell was the most well supported of the three.

**Comment by kashifm-msft** [comment]
ECC cert support for Azure Key Vault is available starting with Azure PowerShell v3.3.0.
